src/Oracle/Oracle.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>3.0.0-preview2</Version>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <RootNamespace>Neo.Plugins</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Akka.TestKit" Version="1.4.5" />
+    <PackageReference Include="Akka.TestKit.Xunit2" Version="1.4.5" />
```

Comment:
Why refer to the unit test packages in the project?

Reply:
Removed